### PR TITLE
Stronger sticky shot sound when charged

### PR DESCRIPTION
**Pull request by braem**
Cvar controls which point during the charge a strong sticky shot sound should be played.

https://cdn.discordapp.com/attachments/627220127254839298/755671335685980241/2020-09-15_23-05-56.mp4

### Checklist
- [x] **I have thoroughly tested all of the code I have modified/added/removed to ensure something else did not break**
- [x] If there is a localization token change, I have updated the `momentum_english_ref.res` file with the changes, ran `tokenizer.py` to generate an up-to-date localization file, and have committed both the `.res` file changes and the new localization `.txt` file
- [x] If I introduced new h/cpp files, I have added them to the appropriate project's VPC file (`server_momentum.vpc` / `client_momentum.vpc` / etc)
- [x] If I have added or modified any visual assets (models, materials, panels, effects, etc), I have taken screenshots / videos of them and attached them to this PR directly (screenshots uploaded through github, videos uploaded to youtube and linked)
- [x] If I have modified any console command, console variable, or momentum entity, I have opened an issue (or a PR) for it in the [Momentum Mod documentation repository](https://github.com/momentum-mod/docs)
- [x] My commits are relatively small and scoped to the best of my ability
- [x] My branch has a clear history of changes that can be easy to follow when being reviewed commit-by-commit
- [x] My branch is functionally complete; the only changes to be done will be those potentially requested in code review

### CODE REVIEW

**Comment by Gocnak** [comment]
Remember the docs repo issue for the new cvar

**Comment by braem** [comment]
Should this be added to the settings panel with the other SJ settings?

**Comment by Gocnak** [comment]
Yes

**Comment by braem** [comment]
Added to settings: 
image

Changed wording to use "powerful" instead of "strong". It really does feel powerful.

**Comment by braem** [comment]
Added cvar for toggling sticky firing sound. Also stopped the "chargestop" sound from playing when `mom_sj_sound_charge_enable` is off.

image

File: mp/game/momentum/resource/momentum_english_ref.res [hunk 1]
```diff
@@ -380,6 +380,9 @@
         "MOM_Settings_SJ_Enable_Detonation_Fail_Sound_TT" "Note: if this is disabled and success sound is enabled, success sound will play if partial detonation happens."
         "MOM_Settings_SJ_Enable_Detonation_Success_Sound" "Stickybomb detonation success sound"
         "MOM_Settings_SJ_Enable_Charge_Sound" "Sticky launcher charging sound"
+        "MOM_Settings_SJ_Enable_Shoot_Sound" "Sticky launcher firing sound"
+        "MOM_Settings_SJ_ShotSoundTheshold_Label" "Powerful Stickybomb Shot Sound Threshold"
```
Comment (Gocnak) [inline mp/game/momentum/resource/momentum_english_ref.res:384]: Nothing else is CamelCase here
```suggestion
        "MOM_Settings_SJ_ShotSoundTheshold_Label" "Powerful stickybomb shot sound threshold"
```

**Review by Gocnak — CHANGES_REQUESTED** [review]

**Comment by braem** [comment]
Fixed that up:
image

**Review by Gocnak — APPROVED** [review]

**Pull request merged**